assets/models/system/databricks-dolly-v2-3b/description.md

```diff
@@ -0,0 +1,60 @@
+
+Databricks-dolly-v2-3b is the smallest model in the Dolly v2 family, with 2.8 billion parameters. It is not the most advanced model, but it still shows impressive performance in following instructions that are not typical of the base model. It can generate text, answer questions, summarize articles, extract information, classify sentences, and more  .
+
+Dolly v2 also has two larger and more powerful models:
+
+- Databricks-dolly-v2-12b, with 12 billion parameters and based on Pythia-12b
+- Databricks-dolly-v2-7b, with 6.9 billion parameters and based on Pythia-6.9b
+
+(1) databricks/dolly-v2-3b · Hugging Face. https://huggingface.co/databricks/dolly-v2-3b
+(2) GitHub - databrickslabs/dolly: Databricks’ Dolly, a large language https://github.com/databrickslabs/dolly
+
+
+
+> The above summary was generated using ChatGPT. Review the <a href="https://huggingface.co/databricks/dolly-v2-3b" target="_blank">original model card</a> to understand the data used to train the model, evaluation metrics, license, intended uses, limitations and bias before using the model.
+
+### Inference samples
+
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href="https://aka.ms/azureml-infer-online-sdk-text-generation-dolly" target="_blank">text-generation-online-endpoint-dolly.ipynb</a>|<a href="https://aka.ms/azureml-infer-online-cli-text-generation-dolly" target="_blank">text-generation-online-endpoint-dolly.sh</a>
+Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-text-generation" target="_blank">text-generation-batch-endpoint.ipynb</a>| coming soon
+
+
+
+### Model Evaluation
+
+Task| Use case| Dataset| Python sample (Notebook)| CLI with YAML
+|--|--|--|--|--|
+Text generation | Text generation | <a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank"> cnn_dailymail </a> | <a href="https://aka.ms/azureml-eval-sdk-text-generation/" target="_blank">evaluate-model-text-generation.ipynb</a> | <a href="https://aka.ms/azureml-eval-cli-text-generation/" target="_blank">evaluate-model-text-generation.yml</a>
+
+
+### Finetuning samples
+
+Task|Use case|Dataset|Python sample (Notebook)|CLI with YAML
+|--|--|--|--|--|
+Text Classification|Emotion Detection|<a href="https://huggingface.co/datasets/dair-ai/emotion" target="_blank">Emotion</a>|<a href="https://aka.ms/azureml-ft-sdk-emotion-detection" target="_blank">emotion-detection.ipynb</a>|<a href="https://aka.ms/azureml-ft-cli-emotion-detection" target="_blank">emotion-detection.sh</a>
+
+
+### Sample inputs and outputs (for real-time inference)
+
+```json
+{
+    "input_data": {
+        "input_string": ["My name is John and I am", "Once upon a time,"]
+    }
+}
+```
+
+#### Sample output
+```json
+[
+    {
```

Comment:
responses in run is different. Could you share the source of below data?

![image](https://github.com/Azure/azureml-assets/assets/61145377/46454e54-f451-42ef-9464-a31cdb0d850b)